sql/job_points.sql

```diff
@@ -0,0 +1,248 @@
+/*
+MySQL Data Transfer
+Source Host: localhost
+Source Database: dspdb
+Target Host: localhost
+Target Database: dspdb
+Date: ???
+*/
+
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
looks like appveyor failed because of github giving a 500 error - nothing to do with the sql file (appveyor doesnt even run the sql files, actually)